controllers/suite_test.go

```diff
@@ -31,7 +31,7 @@ import (
 
 	ipamv1 "github.com/metal3-io/ip-address-manager/api/v1alpha1"
 	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
-	clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha4"
+	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
```

Comment:
I don't if it matters, in https://github.com/metal3-io/cluster-api-provider-metal3/pull/352, I saw clusterv1 being replaced by capi